src/components/label/Label.js

```diff
@@ -25,18 +23,10 @@ export default class Label extends PureComponent {
       size,
     };
 
-    const classNames = cx(
-      theme['label'],
-      {
-        [theme['is-inverse']]: inverse,
-      },
-      className,
-    );
-
     const Element = size === 'large' ? TextDisplay : TextBody;
 
     return (
-      <Box element="label" marginBottom={3} className={classNames} {...others}>
+      <Box display="block" element="label" marginBottom={3} className={className} {...others}>
```

Comment:
You can remove the whole `className` prop now and just let it be in the `...others`.